react-client/src/App.jsx

```diff
@@ -115,10 +117,10 @@ class App extends React.Component {
         </SELLER>
         {/* <SPACE></SPACE> */}
         <MORE id="more-module">
-          <MoreFromShop items={this.state.recommendations} images={this.state.images} seller={this.state.seller} />
+          {/* <MoreFromShop items={this.state.recommendations} images={this.state.images} seller={this.state.seller} /> */}
```

Comment:
Was this commented out because it is reliant on another components data? I just want to make sure that commenting it out doesn't break the front end.

Reply:
Yes, it is relying on an endpoint from another service and its main reason for being commented out is because it doesn't check to make sure that props have been passed in before doing work on it, using a .slice() method on an input, so it breaks immediately. Because of this, I just opted to comment it out for now and will take a closer look later / maybe set up the other components locally.